posthog/batch_exports/service.py

```diff
@@ -636,6 +637,12 @@ def sync_batch_export(batch_export: BatchExport, created: bool):
             ),
             id=str(batch_export.id),
             task_queue=BATCH_EXPORTS_TASK_QUEUE,
+            retry_policy=temporalio.common.RetryPolicy(
+                initial_interval=dt.timedelta(seconds=10),
+                maximum_interval=dt.timedelta(seconds=60),
+                maximum_attempts=2,
+                non_retryable_error_types=["ActivityError", "ApplicationError", "CancelledError"],
```

Comment:
Not sure if I need all of these here, some may be missing. I don't want the workflow to retry on anything _except_ `NondeterminismError`.

Reply:
Due to this uncertainty, I am keeping the number of max attempts on 2. Eventually, we could retry forever on `NondeterminismError`. Maybe.